tasks/gulp/compile-assets.js

```diff
@@ -79,7 +79,10 @@ gulp.task('scss:compile', () => {
 // Compile js task for preview ----------
 // --------------------------------------
 gulp.task('js:compile', () => {
-  return gulp.src([configPaths.src + '**/*.js'])
+  return gulp.src([
+    '!' + configPaths.src + '**/*.test.js',
```

Comment:
Can we flip this so that we start out with a whitelist and then blacklist stuff? I imagine the outcome is the same, but I just think it makes the intent clearer.

Reply:
Was keeping consistent with https://github.com/alphagov/govuk-frontend/blob/master/tasks/gulp/compile-components.js#L18

Reply:
Fair enough 👍